bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/OpenThermGatewayHandlerFactory.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
+import org.openhab.binding.openthermgateway.OpenThermGatewayBindingConstants;
+import org.openhab.binding.openthermgateway.handler.OpenThermGatewayHandler;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * The {@link OpenThermGatewayHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Arjen Korevaar - Initial contribution
+ */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.openthermgateway")
```

Comment:
Remove `immediate` we don't set it on binding components.
```suggestion
@Component(service = ThingHandlerFactory.class, configurationPid = "binding.openthermgateway")
```

Reply:
Adjusted